file/filename.cc

```diff
@@ -66,11 +66,24 @@ static std::string MakeFileName(const std::string& name, uint64_t number,
   return name + buf;
 }
 
+static std::string MakeFileName(uint64_t number, const char* suffix) {
+  char buf[100];
+  snprintf(buf, sizeof(buf), "%06llu.%s",
+           static_cast<unsigned long long>(number),
+           suffix);
+  return buf;
+}
```

Comment:
Now two places implement the same logic. One here and one in the function described in 60 to 67. Is it possible to consolidate them? For example the function in 60-67 can call this function and return `name + "/" + ret`?